.changeset/thirty-carrots-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+"pie-docs": patch
+---
+
+[Changed] - solidify foundations page ordering
```

Comment:
What does it mean to solidify the ordering?

Reply:
Two pages had `order: 5`, so I've fixed that conflict
```suggestion
[Fixed] - Conflict in ordering of pages in Foundations section
```